phpBB/styles/prosilver/theme/mentions.css

```diff
@@ -0,0 +1,92 @@
+/* -------------------------------------------------------------- /*
+    $Mentions
+/* -------------------------------------------------------------- */
+
+/* stylelint-disable selector-max-compound-selectors */
+/* stylelint-disable selector-no-qualifying-type */
+
+/* Mention block
+---------------------------------------- */
+.mention {
+	font-weight: bold;
+}
+
+/* Mention dropdown
+---------------------------------------- */
+.mention-container { /* mention-container */
+	text-align: left;
+	border-radius: 2px;
+	position: absolute;
+	z-index: 999;
+	overflow: auto; /* placed here for list to scroll with arrow key press */
+	max-height: 200px;
+	transition: all 0.2s ease;
+}
+
+.mention-container ul { /* mention-list */
+	margin: 0;
+	padding: 0;
+	list-style-type: none;
+}
+
+.mention-media {
+	display: inline-flex;
+	flex-shrink: 0;
+	justify-content: center;
+	align-items: center;
+	margin-right: 8px;
+	margin-left: 0;
+}
+
+.mention-media-avatar {
+	width: 40px;
+	height: 40px;
+}
+
+.mention-media-avatar img { /* TODO: reconsider for the future design with 1:1 ratio images */
+	max-width: 100%;
+	max-height: 100%;
+}
+
+svg { /* TODO: remove it after general normalization */
```

Comment:
this can be removed we should already have this in base.css I think. If not then it needs to go there...